static-assets/components/cstudio-forms/forms-engine.js

```diff
@@ -2114,9 +2114,19 @@ var CStudioForms =
             });
           };
 
+          // If the repeating group that it's being re-rendered has RTE5 controls, remove all of them before cleanup up
```

Comment:
@rart the issue was that when re-rendering the repeating groups all elements `.tox-silver-sink` were being removed (those are related with RTE menus), and those not related with the repeating group being re-rendered were also removed, causing initialized RTEs to miss the elements and not to behave properly.
Using the `.remove()` function of the RTE removes everything related with the editor being removed.